gradoop-flink/pom.xml

```diff
@@ -50,6 +50,14 @@
     <dependencies>
         <!-- Compile dependencies -->
 
+        <!-- https://mvnrepository.com/artifact/com.sun.xml.bind/jaxb-impl -->
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.2.6</version>
```

Comment:
Dependencies are configured in the parent pom file (gradoop/pom.xml). The version is set as a maven property (check the others). In the module itself (i.e. gradoop-flink) you can then leave the version